### PR TITLE
Add test case for #70019

### PR DESCRIPTION
**Pull request by WindowsMEMZ**
Added a test case for #70019.

### CODE REVIEW

File: validation-test/radar/r118770461.swift
Comment (xedin) [inline validation-test/radar/r118770461.swift:7]: This has to be `// expected-warning` to actually check that the warning is produced; and a TODO comment that it should not be once the issue is fixed?

File: validation-test/radar/r118770461.swift
Comment (xedin) [inline validation-test/radar/r118770461.swift:1]: A couple of notes here:

- I think it would be better to move this to test/Concurrency and use `issue70019.swift` as a name because it’s publicly reported.
- This test currently does nothing because it lacks a `// RUN:` command, please take a look how other tests in test/Concurrency set that up with swift-frontend invocation.

**Review by xedin — CHANGES_REQUESTED** [review]

File: validation-test/radar/r118770461.swift
Comment (WindowsMEMZ) [reply validation-test/radar/r118770461.swift]: Sorry I'm new here, but but could tell me does there any document for swift-frontend invocation? I wonder what's the meaning of the texts after `// RUN:` and what command should I fill in it.

**Comment by WindowsMEMZ** [comment]
@swift-ci Please smoke test

File: validation-test/radar/r118770461.swift
Comment (xedin) [reply validation-test/radar/r118770461.swift]: I’d need to take a look regarding documentation but would be able to help you out with RUN command regardless if you want. Meanwhile you can try copying from test/Concurrency/sendable_*.swift

File: validation-test/Concurrency/issue70019.swift
Comment (xedin) [inline validation-test/Concurrency/issue70019.swift:19]: The message you are checking for has to be wrapped in {{}} and ‘:’ is extraneous.

File: validation-test/Concurrency/issue70019.swift
Comment (xedin) [inline validation-test/Concurrency/issue70019.swift:3]: This is an overkill, it’s building an executable test where I think running typechecker is good enough.

File: validation-test/radar/r118770461.swift
Comment (AnthonyLatsis) [reply validation-test/radar/r118770461.swift]: I think Pavel meant `test/Concurrency`, not `validation-test/Concurrency`. I agree this sort of simple test case fits better in the primary test suite. Funnily enough, I have been around contributing for quite some time, and the validation test suite is *such a mixup* that I am still ignorant to its original purpose. I personally make it as a dumping ground for:

* Internal tooling tests
* Complex non-contrived Swift source code that cannot be reduced to focus on a particular code path

Comment (AnthonyLatsis) [reply validation-test/radar/r118770461.swift]: > I wonder what's the meaning of the texts after `// RUN:` 

These are shell commands that define the test together with the remaining contents of the file. Running the test means running all the `RUN:` commands in the file in order. 

> and what command should I fill in it.

https://github.com/apple/swift/blob/main/docs/Testing.md#writing-tests

Comment (xedin) [reply validation-test/radar/r118770461.swift]: Thank you for the doc link, @AnthonyLatsis!

Comment (WindowsMEMZ) [reply validation-test/radar/r118770461.swift]: Thanks! @AnthonyLatsis

**Comment by xedin** [comment]
I away from my computer whole day today and tomorrow but I will take a look on Monday.

**Comment by WindowsMEMZ** [comment]
> I away from my computer whole day today and tomorrow but I will take a look on Monday.

OK, Thanks. By the way, can I review other pull requests like you? I didn't find the way. And I wonder can I see a bug's detail by `rdar://`, or it just for internal?

**Comment by xedin** [comment]
Okay here is what is passing locally for me:

```
// RUN: %target-typecheck-verify-swift -swift-version 5
// RUN: %target-typecheck-verify-swift -swift-version 5 -enable-library-evolution -I %t

// REQUIRES: concurrency

// https://github.com/apple/swift/issues/70019

@usableFromInline
struct Foo { // expected-note {{consider making struct 'Foo' conform to the 'Sendable' protocol}}
  var integer: Int
}

struct Bar: Sendable {
  // FIXME: This warning should only be thrown when it complies with library evolution mode
  // FIXME: If @usableFromInline is removed from Foo the Sendable conformance is synthesized as expected.
  var foo: Foo // expected-warning {{stored property 'foo' of 'Sendable'-conforming struct 'Bar' has non-sendable type 'Foo'}}
}
```

Notice that it has two `// RUN:` commands because your FIXME states that this warning should only be thrown in library evolution mode we need two runs to make sure that the test starts failing as soon as the issue is fixed. 

Also I suggested naming it issueNNN.swift` but the format is actually `issue-NNNN.swift`, sorry about that!

**Comment by xedin** [comment]
> By the way, can I review other pull requests like you?

Sure, feel free to ping me about other PRs.

File: test/Concurrency/issue-70019.swift
Comment (AnthonyLatsis) [inline test/Concurrency/issue-70019.swift:2]: What’s `-I %t` for?

File: test/Concurrency/issue-70019.swift
Comment (AnthonyLatsis) [inline test/Concurrency/issue-70019.swift:14]:
```suggestion
  // FIXME: This warning should only be thrown in library evolution mode
```

File: test/Concurrency/issue-70019.swift
Comment (AnthonyLatsis) [inline test/Concurrency/issue-70019.swift:15]: This seems like it is supposed to be a multiline FIXME, but it reads as 2 individual ones instead, with the second misleadingly describing *expected* behavior at that. I suggest keeping just the first marker.

File: test/Concurrency/issue-70019.swift
Comment (WindowsMEMZ) [reply test/Concurrency/issue-70019.swift:2]: Let's ask @xedin.

**Comment by WindowsMEMZ** [comment]
> Please squash your commits and see to a proper commit message before we go for merge. You might also want to read our [How to Submit Your First Pull Request](https://github.com/apple/swift/blob/ad68322f5751297283aff8b47cb5a9e292da32b9/docs/HowToGuides/FirstPullRequest.md) guidelines.

Thanks, I'll read it.

File: test/Concurrency/issue-70019.swift
Comment (xedin) [reply test/Concurrency/issue-70019.swift:2]: In this case it's not needed, I copied it from `split-file` test without realizing it.

**Review by xedin — APPROVED** [review]
LGTM, thank you! Once `-I %t` is dropped we can land this.

**Review by AnthonyLatsis — APPROVED** [review]
Thanks! Could you use the PR title as the commit message instead? Git wants them in the imperative mood, plus, GitHub conveniently turns hash references, such as #70019, into links.

**Comment by AnthonyLatsis** [comment]
We were talking just about the `-I %t` argument, not the command itself.

**Comment by WindowsMEMZ** [comment]
> We were talking just about the `-I %t` argument, not the command itself.

I'm sorry about that, it's OK now.

**Comment by AnthonyLatsis** [comment]
@swift-ci please smoke test

**Comment by xedin** [comment]
let's run a full test on macOS just in case

**Pull request merged**
